mbq/client/contrib/tests/test_permissions.py

```diff
@@ -0,0 +1,143 @@
+from typing import Dict
+from unittest import TestCase
+from unittest.mock import Mock
+
+from .. import permissions as sut
+
+
+class TestOSCoreClient:
+    def __init__(self, data):
+        self.data = data
+
+    def fetch_permissions(
+        self, person_id: str, org_ref: sut.UUIDType
+    ) -> sut.FetchedPermissionsDoc:
+        result: Dict = {"global": self.data.get("global", [])}
+        result.update({org_ref: self.data.get(org_ref, [])})
+        return result
+
+    def fetch_permissions_for_location(
+        self, person_id: str, location_id: int, location_type: sut.RefType
+    ) -> sut.FetchedPermissionsDoc:
+        result: Dict = {"global": self.data.get("global", [])}
+        key = f"{location_id}:{location_type}"
```

Comment:
I think this matters, our view is returning `{location_type}:{location_id}` e.g. `company:1` not `1:company`

Reply:
Will switch